packages/lib/src/encryption/Encryption.ts

```diff
@@ -272,10 +270,10 @@ export function checkSignature(
     }
 }
 
-export function signWithSignatureKey(message: Message, keys: Keys): string {
+export function signWithSignatureKey(message: any, keys: Keys): string {
```

Comment:
Was it necessary to change the type from `Message` to `any`?

Reply:
I used this method also to sign an object from type Omit<Messaging.Message, 'signature'>. So it's no longer a message. The implementation expects an object of any type that can be stringified later.